theSingularity.js

```diff
@@ -0,0 +1,389 @@
+  `);
+    directionSouth();
+  } else if (directionInput === "east") {
+    console.log(`
+    WELCOME TO THE OUTSIDE OF SUN CITY!
+    THE ONLY CITY WERE FULLY ORGANIC BEINGS RESIDE
+    IT IS A BEAUTIFUL PLACE!
+    BUT IS A FORTRESS. HOW ARE YOU GOING TO GET IN?`);
+    directionEast();
+  } else if (directionInput === "west") {
+    console.log("West then!");
+    console.log("WELCOME TO THE WILD WILD WEST!");
+    console.log("THE CITY OF REJECTS, OUTDATED A.I AND REPTILIANS");
+    console.log("YOU'll COME ACROSS SOME WAYS TO EARN RESOURCES");
+    console.log("BUT FIRST, YOU HAVE TO PROVE YOUR SELF");
+    console.log("MR T-REX, A REPTILIAN IS CHALLENGING YOU TO A FIGHT ");
+    console.log("HE REMAINS THE CHAMPION AND GUARDIAN OF SCRAP CITY!");
+    console.log(
+      "IF YOU WIN, YOU GET ACCESS TO THE CITY. IF YOU LOOSE YOU DON'T ENTER THE CITY"
+    );
+    directionWest();
+  } else console.log("NOT A VALID DIRECTION");
+  goSomewhere();
+};
+
+const directionNorth = () => {
+  let whereInNorth = readlineSync.question(
+    "Where would you like to go now?\n "
+  );
+  if (whereInNorth === "shelter") {
+    console.log("Shhh someone is in there...");
+    console.log(
+      "Scanvanger: 'I think we are going to get a good prize for this one'"
+    );
+    console.log("'look at her fancy clothes and she has plenty of water'");
+    console.log("'Let's take her to the boss'");
+    console.log("Little girl: 'Nooo! Nooo! Help! Help!'");
+
+    let northArray = ["Go inside", "Pick up rock", "Leave"];
+    let northChoices = readlineSync.keyInSelect(northArray, "What to do?");
+
+    if (northArray[northChoices] === "Pick up rock") {
+      console.log(northArray[northChoices]);
+      console.log("Pow! Pow!");
+      console.log(`You saved the girl!`);
+      console.log("Little girl: Thank you for saving my life!");
+      console.log("I have a code to help you get in Sun City");
+      console.log("The code is 132213");
+      console.log("and here is some water. Let's go East!");
+      goSomewhere();
+    } else if (northArray[northChoices] === "Go inside") {
+      console.log(northArray[northChoices]);
+      console.log("You were spotted and have nothing to defend your self");
+      if (readlineSync.keyInYN("Play Again?")) {
+        play();
+      } else console.log("This place is not for everyone!");
+      quitGame();
+    } else if (northArray[northChoices] === "Leave") {
+      console.log(northArray[northChoices]);
+      console.log("lets get out of here!");
+      directionNorth();
+    } else if (northArray[northChoices] === undefined) {
+      quitGame();
+    }
+  } else if (whereInNorth === "GO BACK") {
+    console.log("LET'S GET OUT OF HERE");
+    goSomewhere();
+  } else console.log("NOT A VALID ENTRY. TRY AGAIN");
+  directionNorth();
+};
+
+const directionSouth = () => {
+  let whereInSouth = readlineSync.question(`WHAT TO DO? : `);
+  switch (whereInSouth) {
+    case "cave":
+      console.log(`
```

Comment:
This stuff in the case should be broken into it's own function. We want to keep our functions short and simple. Adding too much logic can make the code harder to read.